internal/pkg/manifest/validate_env.go

```diff
@@ -104,28 +104,34 @@ func (cfg environmentVPCConfig) validateManagedVPC() error {
 	)
 	var exists = struct{}{}
 	for idx, subnet := range cfg.Subnets.Public {
-		if aws.StringValue((*string)(subnet.CIDR)) == "" || aws.StringValue(subnet.AZ) == "" {
-			return fmt.Errorf(`validate "subnets": public[%d] must include "cidr" and "az" fields if the vpc is configured`, idx)
+		if aws.StringValue((*string)(subnet.CIDR)) == "" {
+			return fmt.Errorf(`validate "subnets": public[%d] must include "cidr" if the vpc is configured`, idx)
```

Comment:
can we use https://github.com/aws/copilot-cli/blob/8e6e7a17bb045bba643b2084d8b12b22fca447c8/internal/pkg/manifest/errors.go#L57 ?

Reply:
done!